### DISCUSSION TITLE
rejected accept github student pack

### DISCUSSION THREAD

**Discussion by GamilaCoding** [community/community#53222] [GitHub Education]
the website I try to enroll to get a student informal place and another place to study in it not on GitHub education I submitted it 2 times and get rejected! what should I do?

**Comment by samus-aran** [comment]
Hi @mermaidseaweav  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by samus-aran** [comment]
Hi @mermaidseaweav,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in [multiple places](https://github.com/orgs/community/discussions/53270). As a result, we have hidden the duplicate comments/discussions, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
